Examples/Rendering/ManyRenderWindows/index.js

```diff
@@ -117,42 +141,65 @@ function addRenderWindow() {
     vtkInteractorStyleTrackballCamera.newInstance()
   );
 
+  const button = createRemoveButton();
+  button.addEventListener('click', () => {
+    rootContainer.removeChild(container);
+    mainRenderWindow.removeRenderWindow(renderWindow);
+    mainRenderWindowView.removeNode(renderWindowView);
+    interactor.delete();
+    renderWindowView.delete();
+    renderWindow.delete();
```

Comment:
isn't renderWindow destructor destructing interactor and renderWindowView as well ?

Reply:
No it doesn't do anything except removing all keys from the model and all callbacks (the default macro.js delete function)

Reply:
@floryst @jourdain WDYT, shouldn't `renderWindow` delete its `interactorWindow` and its `views` ?

Reply:
The `renderWindow` doesn't manage the lifecycle of the interactor or the views, only uses them. The user may want to keep the interactor around after a render window is deleted.

What does VTK do?